src/container/AddTask.js

```diff
@@ -0,0 +1,15 @@
+import { connect } from "react-redux";
+
+import { addTask } from 'src/todoSlice/TodoListSlice';
+import AddTaskInput from "src/components/AddTask/AddTask";
+
+const mapStateToProps = (state) => ({
+  Todos: state.todoList.tasks,
```

Comment:
why we are passing tasks to addTask components? We should only pass addTask function to the component